tools/android_lint/lint.xml

```diff
@@ -7,5 +7,13 @@
         compatible all the way back to API 16. See
         https://developer.android.com/studio/write/java8-support.html#supported_features -->
         <ignore regexp="^Try-with-resources requires API level 19"/>
-  </issue>
+    </issue>
+    <issue id="UnknownNullness" severity="ignore" />
```

Comment:
this is also part of the Kotlin interop, no?

Reply:
This one in particular causes something on the order of ~500 errors